src/index.ts

```diff
@@ -8,6 +12,7 @@ export {
   FieldStatus,
   FieldClasses,
   FieldStyles,
+  RevolutCheckoutLocale,
```

Comment:
Why you've renamed Locale → RevolutCheckoutLocale?

Reply:
- A more consistent export convention e.g., similar to `RevolutCheckoutError` ,  `RevolutCheckoutErrorType` `RevolutCheckoutCardField` and `RevolutCheckoutInstance` exports. 

- Particularly prevents implementors from from always renaming in their projects: 

```js 
 import {Locale} from 'react-intl'
 import {Locale as RevolutCheckoutLocale} from '@revolut/checkout'
```

Considering that `Locale` is a very generic name e.g., exported by most translation libs e.g., `react-intl`. We can be more specific as to the "namespace".

Reply:
`RevolutCheckoutError` is special instance of `Error` object, same goes for `RevolutCheckoutCardField` and `RevolutCheckoutInstance`.

I don't have a strong opinion, just wanted to explain the reasoning behind the naming.